Classes/Information/ExtensionInformation.php

```diff
@@ -216,7 +216,17 @@ public function getControllerClassnames(): array
             if ($file === '..') {
                 continue;
             }
-            $controllerClasses[] = pathinfo($file, PATHINFO_FILENAME);
+            // Only include files that end with "Controller.php"
+            if (!str_ends_with($file, 'Controller.php')) {
+                continue;
+            }
+
+            $className = pathinfo($file, PATHINFO_FILENAME);
+
+            // Must be a valid PHP class name and not start with "_"
+            if (preg_match('/^[A-Za-z]w*$/', $className)) {
```

Comment:
what is this w doing here?

Reply:
It used to be [A-Za-z][A-Za-z0-9] and was changed by rector. Could there be a bug in rector?

Reply:
A general check for class names would be great but not part of a bugfix